governance/src/agent.js

```diff
@@ -189,20 +190,35 @@ function provideInitialize(data) {
   return async function initialize() {
     /* eslint-disable no-param-reassign */
     data.config = {
-      developerAbbreviation: agentConfig.developerAbbreviation,
-      protocolName: agentConfig.protocolName,
-      protocolAbbreviation: agentConfig.protocolAbbreviation,
+      developerAbbreviation: config.developerAbbreviation,
+      protocolName: config.protocolName,
+      protocolAbbreviation: config.protocolAbbreviation,
     };
-    data.address = agentConfig.governance.address;
+    data.goverance = config.contracts;
+    data.contracts = Object.entries(data.goverance).map(([name, entry]) => {
+      const { abiFile } = entry.governance;
+      const { address } = entry;
```

Comment:
```suggestion
      const { governance: { abiFile }, address } = entry.governance;
```

Reply:
changed it to `const { governance: { abiFile }, address } = entry;`